README.md

```diff
@@ -3,5 +3,7 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/4b0a3366dcff40f7be1ad3bf0fe24d30)](https://app.codacy.com/gh/blakes24/dive-atlas?utm_source=github.com&utm_medium=referral&utm_content=blakes24/dive-atlas&utm_campaign=Badge_Grade_Settings)
 [![Coverage Status](https://coveralls.io/repos/github/blakes24/dive-atlas/badge.svg?branch=main)](https://coveralls.io/github/blakes24/dive-atlas?branch=main)
 
```

Comment:
@blakes24 remove this badge since it is no longer needed.

Reply:
This badge is for the main branch, the other badge just shows the coverage for this branch. The main badge should update once there's a Travis CI build on the main branch. Because there's no way of testing that until these changes are applied to the main branch, I was using the `branch=fix-badge` badge to test the changes on this branch. Should I still remove the main branch badge?

Reply:
No...Let me merge and we proceed accordingly.